charts/garden-shoot-trust-configurator/charts/runtime/templates/service.yaml

```
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "garden-shoot-trust-configurator.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "labels" . | indent 4 }}
+  {{- if .Values.additionalAnnotations.service }}
+  annotations:
+    {{- toYaml .Values.additionalAnnotations.service | trim | nindent 4 }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  selector:
+{{ include "labels" . | indent 6 }}
+  ports:
+    - name: health
+      port: 8081
```

Shouldn't this also be configurable?
```suggestion
      port: {{ .Values.config.server.healthPort }}
```